core/src/main/java/tc/oc/pgm/filters/FilterModule.java

```diff
@@ -95,6 +96,7 @@ public FilterModule parse(MapFactory factory, Logger logger, Document doc)
         features.addFeature(null, "sprinting", PlayerMovementFilter.SPRINTING);
         features.addFeature(null, "grounded", GroundedFilter.INSTANCE);
         features.addFeature(null, "flying", FlyingFilter.INSTANCE);
+        features.addFeature(null, "gliding", GlidingFilter.INSTANCE);
```

Comment:
cannot be added here without a proto bump, otherwise any map who was using this as an id for a map feature will break